.github/workflows/test.yml

```diff
@@ -0,0 +1,13 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
as it's said on https://github.com/stefanoeb/jest-action
you don't need this use standard npm install and npm test.